controllers/devworkspace/util.go

```diff
@@ -0,0 +1,26 @@
+package devworkspace
+
+import (
+	chev2alpha1 "github.com/eclipse-che/che-operator/api/v2alpha1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+type DevworkspaceState int
```

Comment:
```suggestion
type DevWorkspaceState int
```
it would be a consistent naming I think, but that's not critical

Reply:
Hmm, from looking at CO codebase, we're consistently "Devworkspace" :) So I'd like to keep it out of this PR and deal with it in another issue. But you're of course right, we should use the same terminology...